### PR TITLE
Draft NL-RSE survey 2023 results

### PR DESCRIPTION
**Pull request by tpronk**
Here is a draft of the NL-RSE survey 2023 results

### CODE REVIEW

File: content/posts/2023-05-25-rse-survey-2023-results.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+---
+
+By Lieke de Boer and Thomas Pronk, NL-RSE
+
+*How are Dutch RSEs doing in 2022/2023? We've sent around a brief survey to find out, which was completed by 23 people. Read on to learn how well RSE interests are safeguarded, what RSEs actually do, and what NL-RSE meetups should look like.*
+<!--break-->
+
+## How (well) are the interests of RSEs safeguarded at their institutions?
+
+The histogram below shows how many participants agreed with the following statement: "The interests of RSEs are safeguarded at my institute". As A substantial number of RSEs agreed with this statement, but there are still places where this seems not to be the case at all.
```
Comment (c-martinez) [inline content/posts/2023-05-25-rse-survey-2023-results.md:15]:
```suggestion
The histogram below shows how many participants agreed with the following statement: "The interests of RSEs are safeguarded at my institute". As a substantial number of RSEs agreed with this statement, but there are still places where this seems not to be the case at all.
```

**Comment by c-martinez** [comment]
Hey @tpronk !

Looks good, I've left a couple of suggestions, but they are fairly trivial. @liekelotte, you do a lot more with blogs, what do you think ? Any suggestions for improvements?

File: content/posts/2023-05-25-rse-survey-2023-results.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+It seems that a lot of progress have been made over the past years. However, there is more to be done. All but one participant agreed that NL-RSE should represent the interests of RSEs nation-wide and so we shall keep representing you. To get an idea how we could do so best, we  asked how interests are safeguarded (or not). Some aspects that RSEs where positive (or negative) about, were:
+* Opportunities for training and career development
+* Recognition of the importance of research software to science
+* Recognition of research software as scientific output in its own right
+* The degree to which RSEs felt isolated or connected in institution-wide networks
+
+## What do RSEs do?
+
+The histograms below show what kind of activities the work of RSEs entails. As one could expect, many RSEs perform engineering work, while a subtantial number also educate people. A lot of variation could be found in the number of RSEs that do science and/or management. It appers that the work of an RSE encompasses much more than just computer programming, reaffirming that research software engineering is a unique discipline in its own right.
```
[the remark this answers is not in the document]
Comment (tpronk) [reply content/posts/2023-05-25-rse-survey-2023-results.md:27]: Check. I'll play a bit with ggplot2 and tweak the labels

**Comment by liekelotte** [comment]
hi! Could we jazz up the images a bit so they don't look like base R? :) 

If you send @c-martinez the data, he said he can make some nice images in Python xkcd style!

**Comment by c-martinez** [comment]
https://matplotlib.org/stable/gallery/showcase/xkcd.html#sphx-glr-gallery-showcase-xkcd-py

**Comment by tpronk** [comment]
Sure! All the data is in long format already for ggplot, so I guess it easily translates. Let me set it up...

File: content/posts/2023-05-25-rse-survey-2023-results.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+## How would you like to take part in NL-RSE meetups?
+
+Only two participants preferred online meetups, while all others preferred hybrid or in-person meetups. 
+
+## Conclusion
+
+**We will keep representing the interests of RSEs.** In particular, we hope to learn from institutions where things are doing well. We will bring their lessons to places where things can improve, especially with regard to training, career development, recognition, and community. 
+
+**We will keep organizing interesting meetups,** with topics that inspire you, increase your technical knowledge, and offer networking opportunities. The meetups will be in-person, ideally with a hybrid option. 
+
```
Comment (c-martinez) [inline content/posts/2023-05-25-rse-survey-2023-results.md:48]:
```suggestion

Code for figures in this blog post can be found [here](https://github.com/c-martinez/nlrse-survey-figures).

```

File: content/posts/2023-05-25-rse-survey-2023-results.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ image: '/img/survey2023/thumbnail.jpg'
 date: 2023-05-25
 ---
 
-By Lieke de Boer and Thomas Pronk, NL-RSE
```
Comment (c-martinez) [inline content/posts/2023-05-25-rse-survey-2023-results.md]: @liekelotte -- very nice of you to include me as an author, but I think you've been pushing this blog post more than I have and deserve the recognition for it!

**Comment by liekelotte** [comment]
I think this can be merged! Looks great. The only thing is - now is not the best time for publicizing. Do we merge when we publicize? What do you think?

**Comment by c-martinez** [comment]
Merge now, publicize later?

**Comment by tpronk** [comment]
No preference

**Comment by tpronk** [comment]
@c-martinez, there seems to be a little issue with the graph "Are interests RSEs Safeguarded". The ticks on the x-axis all say "ree"

**Pull request merged**